server/src/stateManagement.mjs

```diff
@@ -731,6 +683,50 @@ function deleteScratch(userId, key, scope=""){
   }
 }
 
+/* 
+* @function:isUserExist()
+* @Description: check whether the user/appid/group is already exist
+* @Params:userId,users
+* @Return: JSON object {isSuccess: true/false, msg : ``}
+*/
+
+function isUserExist(users,userId){
```

Comment:
Let's rename to "doesUserExist" just to make it flow a little better in the code.

Reply:
Addressed